CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1118,7 +1118,7 @@
         <note/>
       </trans-unit>
       <trans-unit id="Save %@" xml:space="preserve">
-        <source>Save %@</source>
+        <source>Save %@</source><target>Spare %@</target>
```

Comment:
Still dunno where "Yearly Subscription" (just in front of Save 17 %) is coming from. We have "Yearly" in the file, but that's seemingly a whole different string...

Reply:
You're right; I need to change `Yearly Subscription` in App Store Connect → `Yearly` for consistency's sake
<img width="658" alt="Screenshot 2024-05-14 at 5 54 15 PM" src="https://github.com/cliss/callsheet-localizations/assets/282460/7e48d92c-e538-4421-a56d-533314e56a02">

Reply:
Aaaaah, now I get it